plugins/inputs/icinga2/sample.conf

```diff
@@ -2,10 +2,17 @@
 [[inputs.icinga2]]
   ## Required Icinga2 server address
   # server = "https://localhost:5665"
-  
-  ## Required Icinga2 object type ("services" or "hosts")
+
+  ## Collected Icinga2 objects ("services", "hosts")
+  ## Specify at least one object to collect from /v1/objects endpoint.
+  # objects = ["services"]
   # object_type = "services"
```

Comment:
Now you have the old and the new parameter in the sample config?

Reply:
Yeah, that shouldn't be there.